Gems/Atom/RPI/Code/Source/RPI.Public/Pass/Specific/ImageAttachmentPreviewPass.cpp

```diff
@@ -176,14 +176,25 @@ namespace AZ
                 RenderPipeline* pipeline = pass->GetRenderPipeline();
                 if (pipeline)
                 {
-                    for (const auto& binding : pipeline->GetRootPass()->GetAttachmentBindings())
+                    Pass* pipelinePass = pipeline->GetRootPass().get();
+                    PassAttachmentBinding* binding = nullptr;
+
+                    // Get either first output or first input/output
+                    if (pipelinePass->GetOutputCount() > 0)
```

Comment:
We need the renderTarget check here. The output attachment should be a color attachment.